AK/JsonObject.h

```diff
@@ -78,6 +78,23 @@ class JsonObject {
         return m_members.contains(key);
     }
 
+    bool has_null(String const& key) const { return has(key) && get_ptr(key)->is_null(); }
+    bool has_bool(String const& key) const { return has(key) && get_ptr(key)->is_bool(); }
+    bool has_string(String const& key) const { return has(key) && get_ptr(key)->is_string(); }
+    bool has_i32(String const& key) const { return has(key) && get_ptr(key)->is_i32(); }
+    bool has_u32(String const& key) const { return has(key) && get_ptr(key)->is_u32(); }
+    bool has_i64(String const& key) const { return has(key) && get_ptr(key)->is_i64(); }
+    bool has_u64(String const& key) const { return has(key) && get_ptr(key)->is_u64(); }
+    bool has_number(String const& key) const { return has(key) && get_ptr(key)->is_number(); }
+    bool has_array(String const& key) const { return has(key) && get_ptr(key)->is_array(); }
+    bool has_object(String const& key) const { return has(key) && get_ptr(key)->is_object(); }
+#ifndef KERNEL
+    bool has_double(String const& key) const
+    {
+        return has(key) && get_ptr(key)->is_double();
+    }
+#endif
```

Comment:
All of these expand to two hash lookups, one for `has()` and another for `get_ptr()`.
Seems like it would be strictly better to call `get_ptr()` directly in all cases? :)